bookwyrm/templates/ostatus/error.html

```diff
@@ -0,0 +1,60 @@
+{% load i18n %}
+
+{% block content %}
+<div class="block">
+    {% if not request.user.is_authenticated and not error == 'remote_subscribe' %}
+    <div class="navbar-item">
```

Comment:
From what I gather we're not in the main navbar, so I think we can do without this `div`